token/cli/src/main.rs

```diff
@@ -1916,6 +1919,104 @@ async fn command_sync_native(
     })
 }
 
+// Both enable_required_transfer_mesos and disable_required_transfer_mesos
+// Switches with enable_memos bool
+async fn command_required_transfer_memos(
+    token_account_address: Pubkey,
+    owner: Pubkey,
+    bulk_signers: BulkSigners,
+    enable_memos: bool,
```

Comment:
nit: can just call this `enable`

Reply:
haha he had that before and i made the opposite suggestion

Reply:
Bah sorry, I hoped that wasn't the case. I'll retract this and go with yours